content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -19,9 +19,15 @@ Using the latest compatible version of kubectl helps avoid unforeseen issues.
 
 The following methods exist for installing kubectl on Linux:
 
-- [Install kubectl binary with curl on Linux](#install-kubectl-binary-with-curl-on-linux)
-- [Install using native package management](#install-using-native-package-management)
-- [Install using other package management](#install-using-other-package-management)
+- [Install kubectl on Linux](#install-kubectl-on-linux)
+  - [Install kubectl binary with curl on Linux](#install-kubectl-binary-with-curl-on-linux)
+  - [Install using native package management](#install-using-native-package-management)
+  - [Install using other package management](#install-using-other-package-management)
+- [Verify kubectl configuration](#verify-kubectl-configuration)
+- [Optional kubectl configurations and plugins](#optional-kubectl-configurations-and-plugins)
+  - [Enable shell autocompletion](#enable-shell-autocompletion)
+  - [Install `kubectl convert` plugin](#install-kubectl-convert-plugin)
+- [{{% heading "whatsnext" %}}](#-heading-whatsnext-)
```

Comment:
No. This is not a list for "methods for installing kubectl on Linux", the revision makes it a TOC.